wallet-api/src/Ledger/Validation.hs

```diff
@@ -182,8 +195,81 @@ plcDigest = serialise
 -- | Blockchain height
 --   TODO: Use [[Ledger.Height]] when Integer is supported
 data Height = Height
-  { getHeight :: Int
-  } deriving (Eq, Ord, Show, Generic, FromJSON, ToJSON, ToSchema)
+    { getHeight :: Int
+    } deriving (Eq, Ord, Show, Generic, FromJSON, ToJSON, ToSchema)
+
+
+-- | Check if a transaction was signed by a public key
```

Comment:
I mildly preferred these in a separate module, but not a big deal.

Reply:
I wanted to avoid adding another `TH.hs` file if possible

Reply:
Well, in that they previously lived in `Language.PlutusTx.Validation`. On reflection I think it's fine for not all plutus-tx-y stuff to live under `Language.PlutusTx`.